### PR TITLE
add multi-realm example

### PR DESCRIPTION
**Pull request by pkoryzna**

### CODE REVIEW

File: shiro.ini.multi-realm-example [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[main]
+securityManager.subjectDAO.sessionStorageEvaluator.sessionStorageEnabled = false
+
+; This is using DefaultLdapRealm shipped with Shiro.
+; It'll replace {0} placeholder with username supplied via SASL when connecting to Cassandra
+; and try bind with that DN and supplied password.
+
+ldapRealm = org.apache.shiro.realm.ldap.DefaultLdapRealm
+ldapRealm.userDnTemplate = cn={0},ou=users,dc=my-company,dc=com
```
Comment (alex-vv) [inline shiro.ini.multi-realm-example:9]: why placeholder is with zero here and without zero in `shiro.ini.example`?

Comment (pkoryzna) [reply shiro.ini.multi-realm-example:9]: This is the config for the stock Shiro realm - should I change our realm to use `{0]` too?

Comment (alex-vv) [reply shiro.ini.multi-realm-example:9]: I would say yes, to make it consistent, unless there are reasons not to.

Comment (juwalter) [reply shiro.ini.multi-realm-example:9]: 👍

**Review by juwalter — APPROVED** [review]

**Review by juwalter — APPROVED** [review]

**Pull request merged**